fe/fe-common/src/main/java/org/apache/doris/common/LdapConfig.java

```diff
@@ -26,7 +26,7 @@ public class LdapConfig extends ConfigBase {
      * Flag to enable LDAP authentication.
      */
     @ConfigBase.ConfField
-    public static boolean ldap_authentication_enabled = false;
+    public static boolean ldap_authentication_enabled = Config.authentication_type.equalsIgnoreCase("ldap");
```

Comment:
I think this config should be removed direclty?

Reply:
You should remove this config and rebase latest master branch.

Reply:
Modified